conneg-by-ap/index.html

```diff
@@ -459,9 +471,21 @@ <h2>Abstract Model</h2>
       implementation it within specific environments.
     </p>
     <p>
-      Implementations of this abstract model for different environments are called <em>functional <a>profiles</a></em> and to be a valid functional profile they MUST implement these abstract model functions. How they do this will be
+      Implementations of this Abstract Model for different environments are called <em>Functional <a>Profiles</a></em> and to be a valid functional profile they MUST implement these abstract model functions. How they do this will be
```

Comment:
Looking at the full text, I think we'd need a separate definition for "Functional Profile", too.

Reply:
Changed as requested